Lib/profiling/sampling/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Statistical sampling profiler for Python.
+
+This module provides low-overhead profiling by periodically sampling the
+call stack rather than tracing every function call.
+"""
+
+from .collector import Collector
+from .pstats_collector import PstatsCollector
+from .stack_collector import CollapsedStackCollector
```

Comment:
PEP 8 recommends absolute imports:
```suggestion
from profiling.collector import Collector
from profiling.pstats_collector import PstatsCollector
from profiling.stack_collector import CollapsedStackCollector
```